Structure_Engine/Query/Normal.cs

```diff
@@ -36,16 +36,22 @@ public static partial class Query
         /**** Public Methods                            ****/
         /***************************************************/
 
-        [Description("Returns the bars local Z-axis")]
+        [Description("Returns the bars local z-axis, generally the major axis direction of the section of the Bar. \n" +
+                     "For non - vertical members the local z is aligned with the global z and rotated with the orientation angle around the local x. \n" +
+                     "For vertical members the local y is aligned with the global y and rotated with the orientation angle around the local x. For this case the normal will be the vector orthogonal to the local y and local x")]
         [Input("bar", "The Bar to evaluate the normal of")]
-        [Output("normal", "Vector representing the bars local Z-axis")]
+        [Output("normal", "Vector representing the local z-axis of the Bar")]
```

Comment:
Z should be capital for global
z should be lowercase for local

Reply:
I think I changed it to local here? Or am I missing something?

Reply:
In the Description there's instances of global z, also I think we should be saying local z-axis rather than local z, local y.

Reply:
You are right ofc, missed that. Just looked on the output row where I already had changed from capital to lowercase just as you requested and got confused.